spec/rails_helper.rb

```diff
@@ -177,6 +177,16 @@ def read_csv_from_zip(zip_file, filename)
     TestSuiteTimeMachine.travel_permanently_to(CycleTimetable.apply_opens + 1.day)
   end
 
+  config.before(:each, :continuous_applications) do |example|
+    if example.metadata[:continuous_applications].present?
+      FeatureFlag.activate(:continuous_applications)
+      # next recruitment cycle
+      set_time(Time.zone.local(2023, 11, 11, 11))
```

Comment:
What does this date symbolise? Could we use one of the `CycleTimetable` methods instead?

Reply:
I just put a random date. Is there an easy way to say `CycleTimetable.next_year.mid_cycle_date` or something?